Gruntfile.js

```diff
@@ -154,15 +154,58 @@ module.exports = function (grunt) {
     preloadSpreadsheet: {
       'v0.2': 'https://docs.google.com/spreadsheets/d/1ughCy983eK-SPIcDYPsjOitVZzY10WdI2MGGrmxzxF4/pubhtml',
     },
-    buildSwf: {
-      dev: {
-        swfPath: "./.tmp/storage.swf",
-        asPath: "./jsflash/dev/Storage.as"
+
+    mxmlc: {
+        options: {
+          // Task-specific options go here.
+        },
+        dev: {
+          files:{
+            './.tmp/storage.swf' : ['./jsflash/dev/Storage.as']
+          }
+        },
+        prod: {
+          files:{
+            './.tmp/storage.swf' : ['./jsflash/prod/Storage.as']
```

Comment:
prod built to `./dist` before this change, not to `./.tmp`. would've broken saved games for lots of kongregate users if I didn't catch this; careful!